examples/flock.jl

```diff
@@ -53,12 +53,12 @@ function initialize_model(;
     extent = (100, 100),
     seed = 42,
 )
-    space2d = ContinuousSpace(extent; spacing = visual_distance/1.5)
+    space2d = ContinuousSpace(extent; spacing = visual_distance / 1.5)
     rng = Random.MersenneTwister(seed)
 
-    model = StandardABM(Bird, space2d; rng, agent_step!, scheduler = Schedulers.Randomly())
+    model = StandardABM(Bird, space2d; rng, agent_step!, container = Vector, scheduler = Schedulers.Randomly())
     for _ in 1:n_birds
-        vel = rand(abmrng(model), SVector{2}) * 2 .- 1
+        vel = SVector{2}(rand(abmrng(model)) * 2 - 1 for _ in 1:2)
```

Comment:
```suggestion
        vel = rand(abmrng(model), SVector{2}) * 2 .- 1
```

the previous version was more readable, and shouldn't have any impact on performance